_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,4 @@
+{{ site.data.products.db }} clusters support two types of backups:
```

Comment:
nit: I think the word 'clsuters' will cause confusion here, recommend removing it and leaving it as "CockroachDB cloud supports two types of ... "